VRDR/IJEMortality.cs

```diff
@@ -2568,40 +2568,40 @@ public string EAC
         {
             get
             {
-                Tuple<string, string, string, string>[] eac = record.EntityAxisCauseOfDeath;
                 string eacStr = "";
-                foreach(Tuple<string, string, string, string> entry in eac)
+                foreach(var entry in record.EntityAxisCauseOfDeath)
```

Comment:
Is there a project standard/convention to use `var` rather than an explicit type? It makes it less clear what the thing is and what properties/attributes/methods it can be excepted to have.

Reply:
I changed the code to be more explicit